### ISSUE TITLE
Make the Python stream manager client use a context manager

### ISSUE THREAD

**Issue by timmattison** [aws-greengrass/aws-greengrass-stream-manager-sdk-python#5]
Currently the sample code for using stream manager with Python is wrapped in a try with a finally block that cleans up the client. It would be more Pythonic if the client used (Implemented? Contained? I'm not a Python programmer) a context manager so it could be managed automatically and used in a `with` block.

Instead of this:

```
client = None

try:
  client = StreamManagerClient()
  ...
finally:
  if client is not None:
    client.close()
```

It could be:

```
with StreamManagerClient as client:
  ...
```

If this already works this way please let me know. I'd love to clean up my code a bit.

**Comment by sushantAtAws** [comment]
Thanks for the suggestion Tim. We will validate this at our end and update.

**Comment by MikeDombo** [comment]
Released in v1.1.1
